app/controllers/books_controller.rb

```diff
@@ -0,0 +1,10 @@
+class BooksController < ApplicationController
+
+  def show
+    binding.pry
+    @book = Book.find(params[:id])
+    @average_ratings = Book.average_ratings
+    @book_average_rating = @average_ratings.find(params[:id]).avg_rating
+  end
```

Comment:
I think you may have had some confusion around what the spec was asking for. A book has many reviews, so I'd expect to scope from @book.average_rating and I would put an instance method on Book.